pkgs/applications/misc/logseq/default.nix

```diff
@@ -3,8 +3,7 @@
 , fetchurl
 , appimageTools
 , makeWrapper
-# graphs will not sync without matching upstream's major electron version
-, electron_25
+, electron
```

Comment:
Is the comment still correct? If so we should probably keep it and continue to track the major version.

Reply:
Since I don't utilize the syncing feature, I am unsure.

Reply:
@hall Since you wrote the original comment, perhaps you can verify this for us?

Reply:
@SuperSandro2000 It looks like this was changed to `electron_27`. Is this now mergeable?

Reply:
Checked that upstream moved to Electron 27: https://github.com/logseq/logseq/commit/de13df47229656d9996c9e28c5323f1b064f86d1